.github/workflows/tests.yml

```diff
@@ -23,14 +23,6 @@ jobs:
         with:
           swift-version: "6.1"
       
-      - name: Cache iOS Platform
-        uses: actions/cache@v4
-        with:
-          path: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform
-          key: ios-platform-${{ runner.os }}-xcode-${{ hashFiles('/Applications/Xcode.app/Contents/version.plist') }}
-          restore-keys: |
-            ios-platform-${{ runner.os }}-xcode-
```

Comment:
not a huge file so easier to redownload, notice that hte cahce doesn't work all the time anyways